ui/components/app/signature-request-siwe/index.scss

```diff
@@ -0,0 +1,74 @@
+@import 'signature-request-siwe-header/index';
+@import 'signature-request-siwe-message/index';
+
+.signature-request-siwe {
+  display: flex;
```

Comment:
[REQ] I know that you largely just followed the other confirmations patterns but I think it would be good to get this confirmation into shape with new patterns such that we aren't furthering our debt in refactoring / updating these.

We introduced the Box component that is one of the basic building blocks of our design component system. A lot of the styles related to positioning and flex properties can be achieved using those components without adding additional css.

Reply:
Because this new page reuses existing components, the css changes are made to make these existing components match the design, instead of touching existing components and having to test the changes across the repo. For all new components, I'll rebuild them with the box components and make the accompanying css changes!